package.json

```diff
@@ -26,6 +26,7 @@
     "eslint-plugin-import": "^2.29.1",
     "eslint-plugin-jsdoc": "^48.5.2",
     "eslint-plugin-prettier": "^5.1.3",
+    "eslint-plugin-require-extensions": "^0.1.3",
```

Comment:
I wish we had a clear process for adding to our software supply chain.

I did a tiny bit of due diligence...

67k downloads/week. Last published 2 years ago. By solana-labs